includes/templates/story-templates/title-page.html

```diff
@@ -0,0 +1,13 @@
+<!-- wp:amp/amp-story-page {"autoAdvanceAfterDuration":0,"backgroundColors":"[{},{\u0022color\u0022:\u0022#313a43\u0022}]","overlayOpacity":40} -->
```

Comment:
This needs to be replaced with meta blocks instead.

Reply:
Done in 8f1a1a166843cb860cbb0608c62a22a840683ae5